kostyor/tests/unit/upgrades/test_engine.py

```diff
@@ -315,107 +280,84 @@ def test_all_in_one_assignment(self):
-                       'version': constants.MITAKA}),
-            mock.call(self.upgrade,
-                      {'name': 'glance-registry',
-                       'version': constants.MITAKA}),
-            mock.call(self.upgrade,
-                      {'name': 'nova-conductor',
-                       'version': constants.MITAKA}),
-            mock.call(self.upgrade,
-                      {'name': 'nova-scheduler',
-                       'version': constants.MITAKA}),
-            mock.call(self.upgrade,
-                      {'name': 'nova-spicehtml5proxy',
-                       'version': constants.MITAKA}),
-            mock.call(self.upgrade,
-                      {'name': 'nova-api',
-                       'version': constants.MITAKA}),
-            mock.call(self.upgrade,
-                      {'name': 'nova-compute',
-                       'version': constants.MITAKA}),
-            mock.call(self.upgrade,
-                      {'name': 'neutron-server',
-                       'version': constants.MITAKA}),
-            mock.call(self.upgrade,
-                      {'name': 'neutron-linuxbridge-agent',
-                       'version': constants.MITAKA}),
-            mock.call(self.upgrade,
-                      {'name': 'neutron-l3-agent',
-                       'version': constants.MITAKA}),
-            mock.call(self.upgrade,
-                      {'name': 'neutron-dhcp-agent',
-                       'version': constants.MITAKA}),
-            mock.call(self.upgrade,
-                      {'name': 'neutron-metering-agent',
-                       'version': constants.MITAKA}),
-            mock.call(self.upgrade,
-                      {'name': 'neutron-metadata-agent',
-                       'version': constants.MITAKA}),
-            mock.call(self.upgrade,
-                      {'name': 'neutron-ns-metadata-proxy',
-                       'version': constants.MITAKA}),
-            mock.call(self.upgrade,
-                      {'name': 'cinder-api',
-                       'version': constants.MITAKA}),
-            mock.call(self.upgrade,
-                      {'name': 'cinder-scheduler',
-                       'version': constants.MITAKA}),
-            mock.call(self.upgrade,
-                      {'name': 'cinder-volume',
-                       'version': constants.MITAKA}),
-            mock.call(self.upgrade,
-                      {'name': 'heat-api',
-                       'version': constants.MITAKA}),
-            mock.call(self.upgrade,
-                      {'name': 'heat-engine',
-                       'version': constants.MITAKA}),
-            mock.call(self.upgrade,
-                      {'name': 'heat-api-cfn',
-                       'version': constants.MITAKA}),
-            mock.call(self.upgrade,
-                      {'name': 'heat-api-cloudwatch',
-                       'version': constants.MITAKA}),
+        expected_start_calls = [
```

Comment:
In case of `test_multinode_assignment` we pass different hosts, so no way for comprehension. I was thinking about  maybe getting rid of `version` in services (as we defined it via mock) and having something like:

```
            mock.call({'name': 'keystone-wsgi-admin'}, [hosts[0]]),
```

this allows us to reduce number of lines and to increase readability. What do you think?